package.json

```diff
@@ -1,22 +1,19 @@
 {
-  "name": "react-training-RoboDex",
+  "name": "react-training-robodex",
   "version": "0.1.0",
   "private": true,
   "devDependencies": {
     "gh-pages": "^0.12.0",
-    "react-scripts": "0.8.4"
+    "react-scripts": "^0.9.5"
   },
   "dependencies": {
-    "fetch": "^1.1.0",
-    "immutable": "^3.8.1",
+    "prop-types": "^15.5.7",
     "react": "^15.4.1",
```

Comment:
Since the prop-types are being used; I think we should bump the react version to 15.5.0.